apps/passport-server/src/routing/routes/pcdIssuanceRoutes.ts

```diff
@@ -1,131 +1,135 @@
   });
 
+  /**
+   * Gets the RSA public key this server is using for its attestations, so that
+   * 3rd parties can verify whether users have proper attestations.
+   */
   app.get("/issue/rsa-public-key", async (req: Request, res: Response) => {
-    try {
-      if (!issuanceService) {
-        throw new Error("issuance service not instantiated");
-      }
-      res.send(issuanceService.getRSAPublicKey());
-    } catch (e) {
-      rollbarService?.reportError(e);
-      logger(e);
-      res.sendStatus(500);
-    }
+    checkIssuanceServiceStarted(issuanceService);
+    const result = issuanceService.getRSAPublicKey();
+    res.send(result satisfies string);
   });
 
+  /**
+   * Gets the EdDSA public key this server is using for its attestations, so that
+   * 3rd parties can verify whether users have proper attestations.
+   */
   app.get("/issue/eddsa-public-key", async (req: Request, res: Response) => {
-    try {
-      if (!issuanceService) {
-        throw new Error("issuance service not instantiated");
-      }
-      res.send(await issuanceService.getEdDSAPublicKey());
-    } catch (e) {
-      rollbarService?.reportError(e);
-      logger(e);
-      res.sendStatus(500);
-    }
-  });
-
-  app.post("/feeds", async (req, res) => {
-    try {
-      if (!issuanceService) {
-        throw new Error("issuance service not instantiated");
-      }
-
-      const request = req.body as FeedRequest;
-      res.json(await issuanceService.handleFeedRequest(request));
-    } catch (e) {
-      rollbarService?.reportError(e);
-      logger(e);
-      res.sendStatus(500);
-    }
+    checkIssuanceServiceStarted(issuanceService);
+    const result = await issuanceService.getEdDSAPublicKey();
+    res.send(result satisfies EDdSAPublicKey);
   });
 
+  /**
+   * Lets the PCDpass client and 3rd parties inspect what feeds are available
+   * for polling on this server.
+   */
   app.get("/feeds", async (req: Request, res: Response) => {
-    try {
-      if (!issuanceService) {
-        throw new Error("issuance service not instantiated");
-      }
+    checkIssuanceServiceStarted(issuanceService);
+    const result = await issuanceService.handleListFeedsRequest(
+      req.body as ListFeedsRequest
+    );
+    res.json(result satisfies ListFeedsResponseValue);
+  });
 
-      const request = req.body as ListFeedsRequest;
-      res.json(await issuanceService.handleListFeedsRequest(request));
-    } catch (e) {
-      rollbarService?.reportError(e);
-      logger(e);
-      res.sendStatus(500);
-    }
+  /**
+   * Lets a PCDpass client (or even a 3rd-party-developed client get PCDs from a
+   * particular feed that this server is hosting.
+   */
+  app.post("/feeds", async (req, res) => {
+    checkIssuanceServiceStarted(issuanceService);
+    const result = await issuanceService.handleFeedRequest(
+      req.body as PollFeedRequest
+    );
+    res.json(result satisfies PollFeedResponseValue);
```

Comment:
One to come back to after this PR, but we also need to be able to return meaningful error codes here, in case of invalid credentials. I'll create an issue for this.